### PR TITLE
feat: Add timeout per individual NPC chat

### PR DESCRIPTION
**Pull request by magicus**
This way multiple confirmationless chats won't stack up

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java [hunk 1]
```diff
@@ -302,18 +305,15 @@ private void renderDialogue(PoseStack poseStack, List<String> currentDialogue, N
...
-            }
+            if (currentDialogue.isEmpty() && confirmationlessDialogues.isEmpty()) return;
 
-            renderDialogue(
-                    poseStack,
-                    NpcDialogueOverlayFeature.this.currentDialogue,
-                    NpcDialogueOverlayFeature.this.dialogueType);
+            List<String> allDialogues = new ArrayList<>(currentDialogue);
+            confirmationlessDialogues.forEach(d -> {
+                allDialogues.add("");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: If there is only 1, there will be a newline?

Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: Yes... If you can write this without it and without making a mess out of it, please suggest. I'd need something like String.join but for a list.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: I dont think anything clearner could happen other than `if (allDialogues.Length > 0 && allDialouges[0].isBlank()) allDialogues.remove(0)`

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: In fact we dont need to check list length as it is checked before

Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: Well that won't work. If there is a normal dialogue, and a single confirmationless dialogue, there need to be a space inbetween. The only case is if there is no normal dialogue, and just a single confirmationless dialogue.

Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java:312]: I fixed it anyway.

**Pull request merged**